src/synapse/api.py

```diff
@@ -18,11 +18,31 @@
 from urllib3.util import Retry
 
 from charm_state import CharmState
-from constants import MJOLNIR_MANAGEMENT_ROOM, MJOLNIR_MEMBERSHIP_ROOM, SYNAPSE_URL
 from user import User
 
 logger = logging.getLogger(__name__)
 
+CHECK_ALIVE_NAME = "synapse-alive"
+CHECK_MJOLNIR_READY_NAME = "synapse-mjolnir-ready"
+CHECK_NGINX_READY_NAME = "synapse-nginx-ready"
+CHECK_READY_NAME = "synapse-ready"
+COMMAND_MIGRATE_CONFIG = "migrate_config"
+SYNAPSE_CONFIG_DIR = "/data"
+MJOLNIR_CONFIG_PATH = f"{SYNAPSE_CONFIG_DIR}/config/production.yaml"
+MJOLNIR_MANAGEMENT_ROOM = "management"
+MJOLNIR_MEMBERSHIP_ROOM = "moderators"
+MJOLNIR_SERVICE_NAME = "mjolnir"
+MJOLNIR_HEALTH_PORT = 7777
```

Comment:
Can you move to a file under the `tests` directory those constants that are only used for tests?

Reply:
I double-checked and most of those constants are used also in `mjolnir.py`, maybe they are also used to deploy mjolnir?.
 
I also made sure that tests-only constants are moved to `tests/unit/conftest.py`

Reply:
If those are used only in `mjolnir.py` and test files, can yo move them to `mjolnir.py`?

Reply:
I did a cleanup of all the constants in https://github.com/canonical/synapse-operator/pull/36/commits/931dfd28cd354872cfc1466bc6d07234fbc06486, I also talked to Amanda to understand which constant does what and moved them to the correct place